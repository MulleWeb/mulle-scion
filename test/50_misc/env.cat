#! /bin/sh

grep -E '^FOO=' "$@"
