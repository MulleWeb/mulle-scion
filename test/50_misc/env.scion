{%
   taskInfo   = [NSTask mulleStringSystemCallWithCommandString:"env"
                                                   environment:@{ "FOO": "VfL Bochum 1848"}]
   output     = taskInfo["standardOutputString"]
   error      = taskInfo["standardErrorString"]
%}
{{ output }}
{% if( [error length]) %}

stderr:
{{ error }}
{% endif %}
