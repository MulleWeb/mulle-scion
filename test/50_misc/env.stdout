FOO=VfL Bochum 1848
